src/components/modal/index.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Modal from 'react-modal';
+import { toggleDisplayChonkyGraph } from '../../actions/graph';
+import goose from '../../static/images/chonky-goose.jpg';
+import './modal.css';
+
+/**
+ * These are the style rules governing the modal
+ */
+const customStyles = {
+  content: {
+    top: '30%',
+    left: '40%',
+    right: 'auto',
+    bottom: 'auto',
+    color: '#FFFFFF',
+    backgroundColor: '#001521',
+    border: 'none'
+  }
+};
+
+export const ChonkyModal = ({
+  nodesNo,
+  edgesNo,
+  onToggleDisplayChonkyGraph
+}) => {
+  return (
+    <div>
+      <Modal
+        isOpen={true}
+        style={customStyles}
+        overlayClassName="overlay"
+        ariaHideApp={false}>
+        <div className="layout">
+          <img src={goose} alt="goose" />
+          <div>
+            <div className="title">
+              Woahhhhh!
```

Comment:
```suggestion

<div className="title">
              Warning!
              <br />
              Your pipeline is too chonky.
            </div>
            <div className="subtitle">
              Your pipeline contains <b>{nodesNo}</b> nodes and <b>{edgesNo}</b>{' '}
              edges.
              <br />
              Use the filters on the left to select tags and elements and display
              a subset of the pipeline, which is less chonky.
            </div>

```

Reply:
Thanks @stichbury ! The wording has been updated as per the latest design.